packages/site/src/preview/AtlantisPreviewViewer.tsx

```diff
@@ -1,32 +1,38 @@
 import { useAtlantisPreview } from "./AtlantisPreviewProvider";
+import { getPlatformForComponentType } from "../utils/componentTypeUtils";
 
 export const AtlantisPreviewViewer = () => {
   const { iframe, iframeMobile, type } = useAtlantisPreview();
 
-  // A few different approaches were tried to clear out the iframe context when
-  // swapping from mobile -> web -> back, but there was always leftover imports that interfered with each other.
-  // So two iframes was a quick solution to avoid a more complicated solution.
-  // If we ever need a 3rd or a 4th iframe we should probably revisit how to properly
-  // clear out a single iframe instead of continuing this pattern.
```

Comment:
I'm curious about if this comment was referring to the extra scrips being added. I should check.

I'm not able to imagine a world where we need a 3rd or 4th iframe.

since these are platform specific (web and mobile) there isn't really another reasonable thing to ever add. I might leave this comment as deleted if I can confirm the first part is resolved.